proto/import_sstpb.proto

```diff
@@ -221,3 +224,19 @@ message WriteResponse {
     Error error = 1;
     repeated SSTMeta metas = 2;
 }
+
+message DuplicateDetectRequest {
+    kvrpcpb.Context context = 1;
+    bytes start_key = 2;
+    // Return only the keys found by scanning, not their values.
+    bytes end_key = 3;
+    bool key_only = 4;
+    // We only check the data whose timestamp is larger than `min_commit_ts`.
+    uint64 min_commit_ts = 5;
```

Comment:
Is it better that also check data including `min_commit_ts` not only `large than`

Reply:
Emmmm. Maybe not include.....

Reply:
explicitly mention that `min_commit_ts` is "excluded"?